gui.js

```diff
@@ -308,6 +308,14 @@ IDE_Morph.prototype.openIn = function (world) {
         }
     }
 
+    // This function returns the value of a parameter given its key
+    function getParameterByName(name) {
```

Comment:
Not that it's a big deal, but isn't there a URL decoder in cloud.js that could be used?